kernel/configs/fragments/x86_64/gpu/gpu.conf

```diff
@@ -0,0 +1,11 @@
+# Support mmconfig PCI config space access.
+# It's used to enable the MMIO access method for PCIe devices.
+CONFIG_PCI_MMCONFIG=y
+
+# Support to load driver module
+CONFIG_MODULES=y
+CONFIG_MODULE_UNLOAD=y
+CONFIG_MODULE_SIG=y
```

Comment:
cc @grahamwhaley @devimc Dont remember if we ran into issues with CONFIG_MODULE_SIG in the past.

Reply:
I was wondering about this..... in theory I think `MODULE_SIG` is probably a good thing, as then you can only load modules actually built for your kernel.... but, it will add some overhead I think.
I was also thinking it would probably hard-tie the exact nvidia module used to the exact kernel you are loading... which, I'm not sure is the intention, or exactly what you don't want.

@Jimmy-Xu - can you tell us why you enabled it?

Reply:
I don't remember, @Jimmy-Xu  is this option really needed?

Reply:
@grahamwhaley @devimc 
There is a parameter **CONFIG_CRYPTO_FIPS** in kernel/configs/fragments/common/crypto.conf.
It depends on `CRYPTO [=y] && (CRYPTO_ANSI_CPRNG [=y] || CRYPTO_DRBG [=n]) && !CRYPTO_MANAGER_DISABLE_TESTS [=n] && (MODULE_SIG [=n] || !MODULES [=y])`
Therefore, if CONFIG_MODULES is enabled, CONFIG_MODULE_SIG is also required.

If CONFIG_CRYPTO_FIPS is removed, CONFIG_MODULE_SIG can also be removed. 

I ran a test and after removing these two parameters the Nvidia GPU worked fine in a kata container.

Is the CONFIG_CRYPTO_FIPS really needed?

Reply:
Oh, I remember now .... @amshinde [added FIPS](https://github.com/kata-containers/packaging/pull/788), which is somewhat entangled with MODULE. There is a conversation over at https://github.com/kata-containers/packaging/issues/891#issuecomment-567868308 where I dug into it...

Reply:
@grahamwhaley No wonder that config looked familiar :) Now that I see the comments on the related issue, I am fine with having CONFIG_MODULE_SIG.